.gitlab-ci.yml

```diff
@@ -343,8 +342,6 @@ build:base:dev:dune:
   interruptible: true
   extends: .auto-use-tags
   script:
-    - cp theories/dune.disabled theories/dune
-    - cp user-contrib/Ltac2/dune.disabled user-contrib/Ltac2/dune
     - dune build -p rocq-runtime,coq-core,rocq-core,coqide-server
```

Comment:
not sure this job (build:base:dev:dune) is still useful

Reply:
I guess the refman may still need a coq configured to _build/default instead of _install_ci